tabify/fecHeatmapTabify.js

```diff
@@ -34,21 +34,22 @@ export default class FecHeatmapTabify {
     
     process(response) {
         const aggregations = response && response.aggregations;
+        const NO_DATA = 'No Data';
         if (!isEmpty(aggregations)) {
             const result = [];
             if (aggregations.date_histo && aggregations.date_histo.buckets) {
                 for (const dateHistoEntry of aggregations.date_histo.buckets) {
-                    const networkLossValue = (dateHistoEntry.NetworkLoss && dateHistoEntry.NetworkLoss.value) || 0.0;
-                    const lossAfterFecValue = (dateHistoEntry.LossAfterFEC && dateHistoEntry.LossAfterFEC.value) || 0.0;
+                    const networkLossValue = (dateHistoEntry.NetworkLoss && dateHistoEntry.NetworkLoss.value) || NO_DATA;
```

Comment:
lets use conditional 
```
const networkLossValue = dateHistoEntry.NetworkLoss ? dateHistoEntry.NetworkLoss.value : NO_DATA;
```
same for other lines

Reply:
Done.